spec/lib/data_update_scripts/remove_collective_noun_from_site_config_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20201229230456_remove_collective_noun_from_site_config.rb",
+)
+
+describe DataUpdateScripts::RemoveCollectiveNounFromSiteConfig do
+  context "when collective_noun and collective_noun_disabled are removed from the Config" do
+    it "does not alter the community_name" do
```

Comment:
Since I've removed the `collective_noun` and `collective_noun_disabled` fields, I thought that testing that after the removal, the community name remains untouched might be worthwhile. Thoughts!?

Reply:
Ignore the comment above, as this will likely change once I add the fields back in and revert some of the changes I've pushed up in my latest commit!